File: social-service/src/main/java/com/eirsteir/coffeewithme/social/SocialServiceApplication.java
package com.eirsteir.coffeewithme.social;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
import org.springframework.context.annotation.ComponentScan;

@EnableEurekaClient
@ComponentScan(
        basePackages = {
                "com.eirsteir.coffeewithme.commons.exception",
                "com.eirsteir.coffeewithme.social"
        })
@SpringBootApplication
public class SocialServiceApplication {

  public static void main(String[] args) {
    SpringApplication.run(SocialServiceApplication.class, args);
  }
}

File: social-service/src/main/kotlin/com/eirsteir/coffeewithme/social/SocialServiceApplication.kt
package com.eirsteir.coffeewithme.social

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

@EnableEurekaClient
@SpringBootApplication
class SocialServiceApplication

fun main(args: Array<String>) {
    runApplication<SocialServiceApplication>( *args)
}
